linterSettings.js

```diff
@@ -25,6 +26,8 @@ define(function (require/*, exports, module*/) {
     function findFile(fileName, filePath, traverse) {
         var deferred = Promise.defer();
 
+        filePath = FileUtils.getDirectoryPath(stripTrailingSlashes(filePath));
```

Comment:
I can try testing without it, but my suspicion from my previous testing is that the `getDirectoryPath` call isn't important -- but the `stripTrailingSlashes` is what actually produces an effect. My current testing situation is one where I find this quite hard to test, so while I can put up a PR for that change, I cannot test the PR.